process-migration/image.yaml

```diff
@@ -0,0 +1,64 @@
+schema_version: 1
+
+name: "rhpam-7/rhpam75-process-migration-openshift"
+description: "Red Hat Process Automation Manager Process Migration 7.5 container image"
+version: "1.0"
+from: "ubi8:8-released"
```

Comment:
couldn't we use the minimal image for this?

Reply:
@ruromero , @spolti : Ruben, I see you changed this in the standalone image (to use minimal), so I assume you'll be doing the same here?

Reply:
Yes